### DISCUSSION TITLE
Ability to customise CSV reports

### DISCUSSION THREAD

**Discussion by aishwaryashikshalokam** [Sunbird-Ed/Community#555] [Enhancement request]
Hello All

Given below is a use case:

Use Case:  There have been requests from different program managers to be able to create different variations of the reports from the data that are being provided to them in the form of CSVs.
Analysis by Program managers may include a combination of different data points that may be available in separate reports. Eg: Currently in observations we have separate reports for  Domain and Criteria AND Question and responses. These data points together in a report will help in deep analysis of the data.  This requirement also came from CBSE programs. Similarly, there can be a lot of such combinations for different resources.
Having the ability to customize the reports will let Program managers choose what data points they want to see in their reports. They will be able to download the customized reports and gain insights accordingly. They will also be able to remove the data points that may be not relevant to them in a certain resource.

This approach will provide more relevant data to them.

Team has prepared a solution document for the same that highlights on the requirements and suggestive approach.

It will be great to hear your thoughts on the same. 

Here is the link: https://docs.google.com/document/d/19udomymEekYSAH2Qaj72M25aumUq4SwFnMKvEGgZhb8/edit?usp=sharing

We can get on the call to discuss the approach and requirements in detail.

Looking forward to hear from you all.

@Prateek-slokam @aks30 @kiranharidas187 @vijiurs @Sanjana-dm @luckynvdu @surendrasinghs @rhwarrier
@kameshbhr @mmanimegalai @SanthoshVasabhaktula @vrayulu @pritamps @karn-nahata @pkranga @Prerna789 @aayushkedia95 @anandp504 @reshmi-nair @pallakartheekreddy @vinukumar-vs @sharathkashyap

**Comment by rhwarrier** [comment]
Combining different datasets for cross-analysis sounds like a straight forward use case for cQube. Do you want to explore that as a possibility in case it will suffice, instead of trying to build additional features for it?  
cc : @surendrasinghs  @Krishnaj20

**Reply by aishwaryashikshalokam** [reply]
Thank you for your response @rhwarrier.

We will see the feasibility on cQube and update back here
